### PR TITLE
Allow deleting multiple options at once

### PR DESCRIPTION
**Pull request by Sidsector9**
Related #220 

Adds the feature to delete multiple options at once by running a foreach loop on the arguments.

### CODE REVIEW

File: src/Option_Command.php [hunk 1]
```diff
@@ -435,22 +435,28 @@ public function update( $args, $assoc_args ) {
...
+	 * <key>...
 	 * : Key for the option.
 	 *
 	 * ## EXAMPLES
 	 *
 	 *     # Delete an option.
 	 *     $ wp option delete my_option
 	 *     Success: Deleted 'my_option' option.
+	 *
+	 *     # Delete multiple options
```
Comment (thrijith) [inline src/Option_Command.php]:
```suggestion
	 *     # Delete multiple options.
```

File: src/Option_Command.php [hunk 2]
```diff
@@ -435,22 +435,28 @@ public function update( $args, $assoc_args ) {
...
 	public function delete( $args ) {
-		list( $key ) = $args;
-
-		if ( !delete_option( $key ) ) {
-			WP_CLI::error( "Could not delete '$key' option. Does it exist?" );
-		} else {
-			WP_CLI::success( "Deleted '$key' option." );
+		foreach ( $args as $arg ) {
+			if ( ! delete_option( $arg ) ) {
+				WP_CLI::warning( "Could not delete '$arg' option. Does it exist?" );
```
Comment (schlessera) [inline src/Option_Command.php]:
```suggestion
				WP_CLI::warning( "Could not delete '{$arg}' option. Does it exist?" );
```

File: src/Option_Command.php [hunk 3]
```diff
@@ -435,22 +435,28 @@ public function update( $args, $assoc_args ) {
...
+			} else {
+				WP_CLI::success( "Deleted '$arg' option." );
```
Comment (schlessera) [inline src/Option_Command.php]:
```suggestion
				WP_CLI::success( "Deleted '{$arg}' option." );
```

**Comment by schlessera** [comment]
Thanks for the PR, @Sidsector9 !

**Pull request merged**